### ISSUE TITLE
Simple demo of file structure inside patch folder.

### ISSUE THREAD

**Issue by subashz** [djb77/APK-Patcher-Lite#1]
In the read me folder, you asked to copy the pre-compile resources in patch folder.
So, what will be the folder structure inside patch.

Is this the right format "patch/SystemUI/res/drawable/logo.png" ? for replacing the logo of the existing system ui app.

**Comment by djb77** [comment]
Yes that's correct. you could also include patch/SystemUI/classes.dex or patch/SystemUI/resources.arsc if desired
